go.mod

```diff
@@ -3,15 +3,22 @@ module knative.dev/kn-plugin-admin
 go 1.14
 
 require (
-	github.com/maximilien/kn-source-pkg v0.4.8
+	github.com/maximilien/kn-source-pkg v0.5.0
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/spf13/cobra v0.0.6
-	github.com/spf13/viper v1.6.2
-	gopkg.in/yaml.v2 v2.2.8
+	github.com/spf13/cobra v1.1.1
+	github.com/spf13/viper v1.7.1
+	gopkg.in/yaml.v2 v2.4.0
 	gotest.tools v2.2.0+incompatible
-	k8s.io/api v0.17.4
-	k8s.io/apimachinery v0.17.4
-	k8s.io/client-go v0.17.4
-	knative.dev/client v0.14.0
-	knative.dev/serving v0.14.0
+	k8s.io/api v0.18.7-rc.0
+	k8s.io/apimachinery v0.18.7-rc.0
+	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
+	knative.dev/client v0.17.0
+	knative.dev/serving v0.17.0
+)
+
+// copy t from knative/client v0.17 go.mod
+replace (
+	k8s.io/api => k8s.io/api v0.17.6
+	k8s.io/apimachinery => k8s.io/apimachinery v0.17.6
+	k8s.io/client-go => k8s.io/client-go v0.17.6
```

Comment:
I suggest we finish dependency bump in #21 and then rebase on it

Reply:
ok, no problem.